source/_components/egardia.markdown

```diff
@@ -16,40 +17,90 @@ The `egardia` platform enables the ability to control an [Egardia](http://egardi
 
 You will need to know the IP of your alarm panel on your local network. Test if you can login to the panel by browsing to the IP address and log in using your Egardia/Woonveilig account.
 
+{% linkable_title Basic configuration %}
 To enable the integration with your alarm panel, add the following lines to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-egardia:
-  host: YOUR_HOST
-  username: YOUR_USERNAME
-  password: YOUR_PASSWORD
+ ```yaml
+    # Example configuration.yaml entry
+    egardia:
+      host: YOUR_HOST
+      username: YOUR_USERNAME
+      password: YOUR_PASSWORD
 ```
 
-Configuration variables:
-
-- **host** (*Required*): The local IP address of the Egardia/Woonveilig alarm panel.
-- **username** (*Required*): Username for the Egardia/Woonveilig account.
-- **password** (*Required*): Password for Egardia/Woonveilig account.
-- **version** (*Optional*): The version of the Egardia system. `GATE-01`, `GATE-02` and `GATE-03` are currently supported. Defaults to `GATE-01`.
-- **port** (*Optional*): The port of the alarm panel. Defaults to 80.
-- **report_server_enabled** (*Optional*): Enable reporting by server. Defaults to `False`.
-- **report_server_port** (*Optional*): Port of the Egardia server. Defaults to 52010.
-- **report_server_codes** list (*Optional*): List of codes for the different states.
+{% configuration %}
+host:
+  description: The local IP address of the Egardia/Woonveilig alarm panel.
+  required: true
+  type: string
+username:
+  description: Username for the Egardia/Woonveilig account.
+  required: true
+  type: string
+password:
+  description: Password for Egardia/Woonveilig account.
+  required: true
+  type: string
+version:
+  description: The version of the Egardia system. `GATE-01`, `GATE-02` and `GATE-03` are currently supported.
+  required: false
+  type: string
+  default: 'GATE-01'
+port:
+  description: The port of the alarm panel.
+  required: false
+  type: int
+  default: 80
+report_server_enabled:
+  description: Enable reporting by server.
+  required: false
+  type: string
+  default: false
+report_server_port:
+  description:  Port of the Egardia server.
+  required: false
+  type: int
+  default: 52010
+report_server_codes:
+  description: List of codes for the different states.
+  required: false
+  type: map
+  keys:
+    arm:
+      description: List of codes for the 'arm' state.
+      required: false
+      type: map
```

Comment:
I think you can specify `list` here, although it's not clearly mentioned in our docs for configuration tag.

Reply:
well, it is *not* mentioned, but let me try :)